libc/config/linux/x86_64/entrypoints.txt

```diff
@@ -429,12 +429,12 @@ set(TARGET_LIBM_ENTRYPOINTS
     libc.src.math.nextafter
     libc.src.math.nextafterf
     libc.src.math.nextafterl
-    libc.src.math.nexttoward
-    libc.src.math.nexttowardf
-    libc.src.math.nexttowardl
     libc.src.math.nextdown
     libc.src.math.nextdownf
     libc.src.math.nextdownl
+    libc.src.math.nexttoward
+    libc.src.math.nexttowardf
+    libc.src.math.nexttowardl
```

Comment:
I just reordered the existing entrypoints for linux/x86_64 here to keep them sorted.